CHMeetupApp/Sources/ViewControllers/Events/Registration/Preview/DisplayModel/RegistrationFormDataCollection.swift

```diff
@@ -70,12 +70,13 @@ final class FormDisplayCollection: NSObject, DisplayCollection, DisplayCollectio
 
   func headerTitle(for section: Int) -> NSAttributedString {
     let cell = formData.sections[section]
-    let attributtedString = NSMutableAttributedString(string: cell.name,
-                                                      attributes: [NSFontAttributeName: DefaultTableHeaderView.font])
+    let attributtedString
+      = NSMutableAttributedString(string: cell.name,
+                                  attributes: [NSAttributedStringKey.font: DefaultTableHeaderView.font])
```

Comment:
attributes: attribute)

Reply:
SwiftLint warning: Line should be 120 characters or less: currently 124 characters